File: Python/leccion9/Persona.py
class Persona:  # Creamos una clase

    def __init__(self, nombre, apellido, dni, edad, *args, **kwargs):  # Se lo llama método Init Dunder
        self.nombre = nombre
        self.apellido = apellido
        self._dni = dni # Este atributo esta encapsulado de una manera sugerida
        self.edad = edad
        self.args = args
        self.wkargs = kwargs

    def mostrar_detalle(self): # self es igual a this
        print(f'La clase Persona tiene los siguientes datos: {self.nombre} {self.apellido} {self._dni} {self.edad}, la dirección es: {self.args}, los datos importantes son: {self.wkargs}')

File: java/training/java_utn/leccion03/ClasesObjetos/src/clasesobjetos/Persona.java

 
package clasesobjetos;

public class Persona {
    //Atributod de la clase
    public String nombre;
    public String apellido;
    String estudios;
    // Metodos de la clase
    public void obtenerInformacion(){
        System.out.println("Nombre: " + nombre);
        System.out.println("Apellido: "+apellido);
        System.out.println("Estudios: "+ estudios);
    }
}
